src/index.ts

```diff
@@ -138,95 +148,390 @@ export = (app: Probot) => {
               context: newContribution.context,
               additionalInfo: newContribution.additionalInfo,
               createdAt: newContribution.createdAt,
-            }
-          ]
-        })
+            },
+          ],
+        });
       }
-      
-      if ('sha' in contributions.data) {
+
+      if ("sha" in contributions.data) {
         await context.octokit.repos.createOrUpdateFileContents({
-          owner: "anyproto",
+          owner: org,
           repo: targetRepo,
           path: targetFile,
           sha: contributions.data.sha,
-          message: "Add @" + newContribution.login + " for " + newContribution.contributionType + " (requested in " + repo + "#" + number + ")",
+          message:
+            "Add @" + newContribution.login + " for " + newContribution.contributionType + " (requested in " + repo + "#" + number + ")",
           content: Buffer.from(JSON.stringify(content, null, 2)).toString("base64"),
           branch: "new-contributors",
-        })
+        });
       } else {
-        throw new Error("Could not get sha of contributors.json")
+        throw new Error("Could not get sha of contributors.json");
       }
 
       // create pull request to merge new-contributors into main
       try {
         await context.octokit.pulls.create({
-          owner: "anyproto",
+          owner: org,
           repo: targetRepo,
           title: "Add new contributions",
           head: "new-contributors",
           base: targetBranch,
           body: "Recognizing new contributions.",
-        })
-      } catch (error) {
+        });
+      } catch (error) {}
+    }
+  });
+
+  // For "🆕 New" issues, change status to “🏗 In progress”, assigne @any-association to the issue, save comment’s author name to “Lead contributor”
+  // command format: @any assign me
+  // For “🏗 In progress” issues, change status to “New”, remove assignee, remove the content of “Lead contributor"
+  // command format: @any unassign me
+  app.on("issue_comment", async (context) => {
+    const comment = context.payload.comment.body.trim();
+    const user = context.payload.comment.user.login;
+    const issue = context.payload.issue;
+    const issueNumber = context.payload.issue.number;
+    const repository = context.payload.repository.name;
+    const words = comment.split(" ");
+
+    if (
```

Comment:
it will be easier to read if you make all common checks in the beginning and then have a switch statement for different commands